per/models.py

```diff
@@ -656,3 +657,32 @@ class PerWorkPlan(models.Model):
 
     def __str__(self):
         return f"{self.overview.id}"
+
+
+class LearningType(models.IntegerChoices):
+    LESSON_LEARNED = 1, _('Lesson learned')
+    CHALLENGE = 2, _('Challenge')
+
+
+@reversion.register()
+class OpsLearning(models.Model):
+    learning = models.TextField(verbose_name=_("learning"), max_length=900, null=True, blank=True)
+    learning_validated = models.TextField(verbose_name=_("learning (validated)"), max_length=900, null=True, blank=True)
+    appeal_code = models.CharField(verbose_name=_("appeal (MDR) code"), max_length=20, null=True, blank=True)
```

Comment:
I wonder if we should maybe have another field called `appeal` that's an actual foreign key to Appeals? This is also fine, just thinking out loud ..

Reply:
Good catch. Should be aligned with @ypyelab .